src/common/regions/region_db.ts

```diff
@@ -88,7 +95,7 @@ class RegionDB {
   }
 
   all(): Region[] {
-    return [...this.states, ...this.counties, ...this.metroAreas];
+    return this.allRegions;
```

Comment:
do you think there's much advantage in having `allRegions` internally but `all()` publicly?  Seems like all could be a property